vertx-consul-client/src/main/java/io/vertx/ext/consul/impl/ConsulClientImpl.java

```diff
@@ -39,6 +39,7 @@
 public class ConsulClientImpl implements ConsulClient {
 
   private static final String TOKEN_HEADER = "X-Consul-Token";
+  private static final String INDEX_HEADER = "X-Consul-Index";
```

Comment:
It might be useful to add some javadoc with page listing / explaining these headers.